packages/datetime2/src/components/date-range-picker3/_date-range-picker3.scss

```diff
@@ -59,6 +59,13 @@
     }
   }
 
+
+  /* stylelint-disable max-line-length */
+  &.#{$ns}-datepicker.#{$ns}-daterangepicker-single-month.#{$ns}-daterangepicker-showing-outside-days .rdp-day_outside {
+    pointer-events: none; // weird interactions happen if allowing outside days to be clicked
```

Comment:
Note that without this, selecting a date in the earlier "outside days" switches to the previous month. This doesn't seem ideal since there will not be many days ahead of that initial date selected once we switch to the previous month, but the user can reselect the month after making the start date selection.

Also note that we only need to worry about click and not keyboard nav events here because we still want to allow keyboard nav with arrow keys to navigate between months. The difference is that keyboard nav is navigation, then selection, while mouse interaction is selection and navigation which felt a bit jarring.

Open to suggestions for a better solution here.

Reply:
I would be curious for @alex-dales's opinion. Playing around with the UI, I actually found it surprising that I couldn't click on the grayed out dates. It might not be obvious to some users that they have to switch to the previous month intentionally to select these "outside day" dates?